docs/cpp/guides/hash.md

```diff
@@ -108,8 +108,9 @@ Of course, this works only if `MyKey` is hashable by `absl::Hash`, i.e.
 `absl::Hash` supports the `MyKey` type.
 
 NOTE: the hash codes computed by `absl::Hash` are not guaranteed to be stable
-across different runs of your program. In fact, in the usual case it randomly
-seeds itself at program startup.
+across different runs of your program, or across different dynamically loaded
+libraries in your program. In fact, in the usual case it randomly seeds itself
+at program startup.
```

Comment:
I would prefer not to comment this. It is true today, but I think the note with this reads as "here's how it works." whereas without this sentence, it reads more as "please don't rely on this." I think we're aiming for the latter.

Reply:
Done.